src/main/java/kr/co/programmers/urlshortner/UrlRepository.java

```diff
@@ -0,0 +1,12 @@
+package kr.co.programmers.urlshortner;
+
+import java.util.Optional;
+import kr.co.programmers.urlshortner.domain.UrlHolder;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UrlRepository extends JpaRepository<UrlHolder, Long> {
```

Comment:
```suggestion
public interface UrlRepository extends JpaRepository<UrlHolder, Long> {
```
마이너하지만, JpaRepository에 repository 어노테이션이 포함되어있어서 안달아도 동작합니다 😶